User/Makefile

```diff
@@ -5,6 +5,8 @@
 
 .DEFAULT_GOAL := all
 
+.ONESHELL:
```

Comment:
One needs this to make "here document" constructions work correctly inside makefiles. Otherwise it tries to interpret `exit-on-time = 7200` as a separate command.